bubble_sort.rb

```diff
@@ -0,0 +1,32 @@
+def bubble_sort_by(array)
+  i = 0
+  length = array.length - 2
+  while i <= length
+    if array[i] > array[i + 1]
+      array[i], array[i + 1] = array[i + 1], array[i]
+      i = -1
+    end
+    i += 1
+  end
+  print array
```

Comment:
Please remove the `print` statement and instead return ` array`. Your method must return the sorted array.